### DISCUSSION TITLE
Is it possible to stop training from web UI?

### DISCUSSION THREAD

**Discussion by quasiblob** [bmaltais/kohya_ss#323] [Ideas]
Is it possible to somehow stop the training from the web UI without killing the whole process from the terminal?

Edit: I know there is the other similar thread - but I only interested about canceling a training because of some incorrect settings etc.

If I've missed this somehow, please let me know, otherwise - it would be really nice if there was a stop / cancel button.

**Comment by bmaltais** [comment]
Not at the moment. Kohya would need to support that 1st in his trainer... then would need to figure out how to send the commands to the trainer...
